packages/web3torrent/notes/channels-integration.md

```diff
@@ -0,0 +1,51 @@
+# How are channels integrated into web3Torrent?
+
+Web3Torrent uses state channels tech to stream money between webTorrent peers, so that downloaders may pay uploaders for data. The granularity of the payments can be tuned, which alters the trust-performance tradeoff. For example, peers can pay per byte (low trust, high overhead for processing payments resulting in low data transfer speeds) or per Megabyte (higher trust, lower overhead and greater transfer speed).
+
+These tradeoffs are set in [`constants.ts`](https://github.com/statechannels/monorepo/blob/master/packages/web3torrent/src/constants.ts).
+
+## Paid streaming extension
+
+We wrote a small extension to the BiTorrent protocol that allows for extra information to exchanged between torrenting peers: sufficient for them to run a state channel.
```

Comment:
It should also be mentioned that this extension is the one in charge of intercepting and blocking and resuming data transfers, to control the flow of data between two peers.